packages/affiliates/libs/bigquery.js

```diff
@@ -12,17 +11,15 @@ module.exports = ({ client } = {}) => {
     logsByContract(contract, start, end = Date.now(), selection = ["*"]) {
       assert(contract, "requires contract");
       assert(start, "requires start");
-      start = moment(start).format("YYYY-MM-DD hh:mm:ss");
-      end = moment(end).format("YYYY-MM-DD hh:mm:ss");
       // require an array of values
       selection = lodash.castArray(selection);
       return `
         SELECT ${selection.join(", ")}
         FROM
           bigquery-public-data.crypto_ethereum.logs
         WHERE
-          block_timestamp >= TIMESTAMP('${start}')
-          AND block_timestamp < TIMESTAMP('${end}')
+          block_timestamp >= TIMESTAMP_MILLIS(${start})
```

Comment:
Can you briefly explain what the issue was? Is this a BQ native function? What are the inputs and outputs?

Reply:
Yes after running the emp rewards script with the same date as liquidity mining, there were really wildy different block numbers returned from each script for start/end blocks. They are both doing block number lookups from time with different techniques. It turns out that the bigquery time was being interpreted as AM, while the LM script was properly looking up by PM.  This was an error in how we were formatting the date/time before submitting it to bigquery. The fix was to remove timestamp conversion to a string format, and use BQ TIMESTAMP_MILLIS to interpret within BQ instead.

Reply:
I see. +1